src/main/java/edu/harvard/iq/dataverse/api/Dataverses.java

```diff
@@ -574,7 +591,76 @@ public Response deleteDataverse(@Context ContainerRequestContext crc, @PathParam
             return ok("Dataverse " + idtf + " deleted");
         }, getRequestUser(crc));
     }
-
+    
+    /**
+     * Endpoint to change attributes of a Dataverse collection.
+     *
+     * @apiNote Example curl command:
+     *          <code>curl -X PUT -d "test" http://localhost:8080/api/dataverses/$ALIAS/attribute/alias</code>
+     *          to change the alias of the collection named $ALIAS to "test".
+     */
+    @PUT
+    @AuthRequired
+    @Path("{identifier}/attribute/{attribute}")
+    public Response updateAttribute(@Context ContainerRequestContext crc, @PathParam("identifier") String identifier,
```

Comment:
I just started working on #8889, enabling file pids in specific collections. It should be much more straightforward than the functionality in this PR.
Is it ok if I just copy-and-paste most of this API into my branch and use it for my new collection setting there?

Reply:
(I added a simple documentation entry for this API in that PR, if you want to pull into your branch and expand on it)

Reply:
> hope that was ok!

That's fine. You could have cherry-picked (so it retain authorship and commit count), but never mind. 🤗